### PR TITLE
IoTConsensusV2: Support Table Model Replicate

### PR DESCRIPTION
**Pull request by Pengzna**
as title.

Note: pipeDeleteNodeEvent remaining adoption for pipe

### CODE REVIEW

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/analyzer/StatementAnalyzer.java [hunk 1]
```diff
@@ -492,7 +492,7 @@ protected Scope visitUse(Use node, Optional<Scope> scope) {
 
     @Override
     protected Scope visitInsert(Insert insert, Optional<Scope> scope) {
-      throw new SemanticException("Insert statement is not supported yet.");
+      throw new SemanticException("in");
```
Comment (SteveYurongSu) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/analyzer/StatementAnalyzer.java:495]: WHAT IS THIS FOR? :)

**Comment by SteveYurongSu** [comment]
SOME UTs FAILED, PLEASE CHECK

**Comment by Pengzna** [comment]
> SOME UTs FAILED, PLEASE CHECK

fixed

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Review by SteveYurongSu — APPROVED** [review]

**Review by jt2594838 — APPROVED** [review]

**Pull request merged**